hack/codegen.sh

```diff
@@ -41,9 +52,9 @@ skugen() {
 
 
 skugen-all() {
-  SUBSCRIPTION_ID=$(az account show --query 'id' --output tsv)
-  export SUBSCRIPTION_ID
-  if [ -z "${SUBSCRIPTION_ID}" ]; then
+  AZURE_SUBSCRIPTION_ID=$(az account show --query 'id' --output tsv)
```

Comment:
Swapped this to `AZURE_SUBSCRIPTION_ID` because it's what we require in Karpenter setup and it felt reasonable to be consistent.